validation/multi_region/cubed_sphere_steady_state.jl

```diff
@@ -89,7 +89,7 @@ model = HydrostaticFreeSurfaceModel(; grid,
 =#
 
 model = HydrostaticFreeSurfaceModel(; grid,
-                                    momentum_advection = VectorInvariant(),
+                                    VectorInvariant(vorticity_scheme = WENO()),
```

Comment:
what about divergence scheme and vertical?

Reply:
Nz=1 here

Reply:
Ok so no vertical, but still divergent motions right?

Reply:
Got it